app/assets/v2/js/pages/new_bounty.js

```diff
@@ -185,6 +186,9 @@ $(document).ready(function() {
     open_hiring_panel(true);
   });
 
+  $('#specialEvent').click(function() {
```

Comment:
update this to `$('#specialEvent').on('click', () =>  {` ? jquery folks are trying retiring the `.click`

Reply:
Actually `open_hiring_panel` before this wasn't even declared I think the function name was changed in some point, and behind the function to open advance options also that area doesn't exist anymore.  Making some clean now